haiku-apps/bepodder/bepodder-1.5.0.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A media aggregator (and more)"
+DESCRIPTION="BePodder helps you to find, subscribe to, and download \
+multimedia contents. It allows you to easily enjoy audio, video, images and \
+news published around the web in RSS format. It's a Podcaster software.
+
+BePodder includes many cool features:
+
+- Easy to use yet powerful graphical user interface
+- Handles all news/podcast feeds in RSS 0.9, 2.0, Atom, RDF and iTunes
+- Fully integrated with Haiku's BFS file system
+- Integrated podcast, videoblog, imagefeeds, BitTorrentfeeds, newsfeeds and \
+Internationalfeeds, built-in OPML directories with thousands of feeds
+- Advanced channels/lists management with drag and drop support
+- Easy to integrate with other applications like BitTorrent / Installer / \
+Unzip / etc.
+- Players integration with a new 'Auto' option
+- BePodder is 100% adwares/spywares free
+- You don't need an iPod or any kind of MP3 player to use and enjoy BePodder
+- Proxy support
+- Automatic channels check
+- Up to 5 enclosure downloads at the same time (multi-threading)"
+HOMEPAGE="https://github.com/HaikuArchives/BePodder"
+COPYRIGHT="2006-2014 Funky Idea Software"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="c63b1ed184c6425ec42f32c08a02c22ea7b3aa8625ef33c6577b5639a9e5b45f"
+SOURCE_DIR="BePodder-$portVersion"
+SOURCE_FILENAME="bepodder-v$portVersion.tar.gz"
```

Comment:
Oops, `SOURCE_FILENAME` should be above `SOURCE_DIR`.
```diff
+SOURCE_FILENAME="bepodder-$portVersion.tar.gz"
 SOURCE_DIR="BePodder-$portVersion"
-SOURCE_FILENAME="bepodder-v$portVersion.tar.gz"
```
([HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering))

BTW, I think `SOURCE_FILENAME` would look nicer without the "v" before `$portVersion.tar.gz`. But this is just a suggestion.

Reply:
Should I use a new tag without the "v"?

Reply:
It's ok so @janus2 no need for a retag (we'll adjust with the next version if need be)
@fbrosson I used it because the actual download also uses it, but it could be left out if you want :)

Reply:
All the software a took as an example has the v BeShare, PecoRename, FilWip, ...

Reply:
No problem @janus2 maybe just look at the versioning from the last release next time :)